src/cata_tiles.cpp

```diff
@@ -1125,7 +1125,7 @@ bool cata_tiles::draw_from_id_string(std::string id, TILE_CATEGORY category,
     }
 
     if (it == tile_ids.end()) {
-        long sym = -1;
+        wchar_t sym = -1;
```

Comment:
If the compiler warned about ```wchar_t < int```, I'm pretty sure this is going to make it angry too.